sdk/python/kfp/client/client.py

```diff
@@ -303,7 +303,7 @@ def _load_config(
                 namespace)
         return config
 
-    def _is_inverse_proxy_host(self, host):
+    def _is_inverse_proxy_host(self, host: str):
```

Comment:
nit: return annotation

Can you scan for this throughout?

Reply:
Done! 👍